lib/std/build.zig

```diff
@@ -2155,11 +2194,23 @@ pub const LibExeObjStep = struct {
             }
         }
 
-        if (self.build_options_contents.items.len > 0 or self.build_options_artifact_args.items.len > 0) {
-            // Render build artifact options at the last minute, now that the path is known.
+        if (self.build_options_contents.items.len > 0 or
+            self.build_options_artifact_args.items.len > 0 or
+            self.build_options_write_file_args.items.len > 0)
+        {
+            const path_option_fmt = "pub const {s}: []const u8 = \"{}\";\n";
```

Comment:
You probably meant to use this somewhere, also remember to use `std.zig.fmtId` and  `std.zig.fmtEscapes.

Reply:
nice catch